# INSTALL_LuMP_Pulsar_ATLAS_P1.patch
# patch file 0 for ATLAS
# 2014 Oct 02  James Anderson  --- GFZ start
#
# Copyright (c) 2014 James M. Anderson <anderson@gfz-potsdam.de>
#
# Licensed under the Apache License, Version 2.0 (the "License");
# you may not use this file except in compliance with the License.
# You may obtain a copy of the License at
#
#     http://www.apache.org/licenses/LICENSE-2.0
#
# Unless required by applicable law or agreed to in writing, software
# distributed under the License is distributed on an "AS IS" BASIS,
# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
# See the License for the specific language governing permissions and
# limitations under the License.
#
#
# Note that the ATLAS software in this patch is presumably covered by a different
# copyright.  See http://math-atlas.sourceforge.net/
# for more information about ATLAS copyright issues.
#
################################################################################
diff -Naur ATLAS3.11.30/Linux/lib/Makefile ATLAS3.11.30_new/Linux/lib/Makefile
--- ATLAS3.11.30/Linux/lib/Makefile	2014-10-02 11:28:20.131571369 +0200
+++ ATLAS3.11.30_new/Linux/lib/Makefile	2014-10-02 11:39:23.384776291 +0200
@@ -58,7 +58,7 @@
            --whole-archive $(libas) --no-whole-archive $(LIBS)
 GCCTRY_WIN:
 	$(GOODGCC) -shared -o $(outso) -Wl,--output-def=$(outdef) \
-           -Wl,"rpath-link $(LIBINSTdir)" \
+           -Wl,-rpath-link,"$(LIBINSTdir)" \
            -Wl,--whole-archive $(libas) -Wl,--no-whole-archive $(LIBS)
 GCCTRY_norp_WIN:
 	$(GOODGCC) -shared -o $(outso) -Wl,--output-def=$(outdef) \
@@ -124,7 +124,7 @@
            --whole-archive $(libas) --no-whole-archive $(LIBS)
 GCCTRY:
 	$(GOODGCC) -shared -o $(outso)  \
-           -Wl,"rpath-link $(LIBINSTdir)" \
+           -Wl,-rpath-link,"$(LIBINSTdir)" \
            -Wl,--whole-archive $(libas) -Wl,--no-whole-archive $(LIBS)
 GCCTRY_norp:
 	$(GOODGCC) -shared -o $(outso)  \
